build.gradle.kts

```diff
@@ -629,6 +638,10 @@ tasks {
   register("checksumTrinoConnector") {
     group = "gravitino distribution"
     dependsOn(assembleTrinoConnector)
+    // To fix gradle assembleDistribution errors: "Task ':checksumTrinoConnector' uses this output
+    // of task ':assembleSparkConnector' without declaring an explicit or implicit dependency", this
+    // will not add depends on assembleSparkConnector.
+    mustRunAfter(assembleSparkConnector)
```

Comment:
Does the task `checksumTrinoConnector` use the output of task `assembleSparkConnector`? so what's the output   
directory or file that `checksumTrinoConnector` uses?

Reply:
`checksumTrinoConnector` doesn't use the output of `assembleSparkConnector` actually.`checksumTrinoConnector` uses the Trino package produced by `assembleTrinoConnector` which shares `distribution` directory.